src/stores/walletStore.spec.ts

```diff
@@ -42,6 +50,36 @@ describe('wallet store', () => {
     expect(wallets).toContainEqual(wallet);
   });
 
+  describe('existingWallet', () => {
+    let wallet: WalletModel;
+
+    beforeEach(async () => {
+      walletStore.clearWallets();
+      const dto = newWalletApiKey();
+      const store = walletStore;
+      wallet = new WalletModel(store, dto);
```

Comment:
as a side note: it's better to instantiate objects with the store, `walletStore.createWallet` in this case

Reply:
agree